commit 65e06b20130da895fda8e3698a7ebe18e14d13ca
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Wed Oct 1 10:06:27 2008 +0300

    Tweaks to how ISA-macros are generated (rhbz#464754)
    - for ppc and sparc, move the check for 64bit version first and use a
      wildcards to cover more field automatically

diff --git a/installplatform b/installplatform
index bb5933d..8861df0 100755
--- a/installplatform
+++ b/installplatform
@@ -80,13 +80,13 @@ for SUBST in $SUBSTS ; do
   ISANAME=
   ISABITS=
   case "${ARCH}" in
-    sparc|sparcv8|sparcv9*) 
+    sparc64*) 
 	ISANAME=sparc
-	ISABITS=32
+	ISABITS=64
 	;;
-    sparc64|sparc64v) 
+    sparc*) 
 	ISANAME=sparc
-	ISABITS=64
+	ISABITS=32
 	;;
     s390)
 	ISANAME=s390
@@ -96,13 +96,13 @@ for SUBST in $SUBSTS ; do
 	ISANAME=s390
 	ISABITS=64
 	;;
-    ppc)
+    ppc64*)
 	ISANAME=ppc
-	ISABITS=32
+	ISABITS=64
 	;;
-    ppc64)
+    ppc*)
 	ISANAME=ppc
-	ISABITS=64
+	ISABITS=32
 	;;
     i?86|pentium?|athlon|geode)
 	ISANAME=x86
